DMTR-102.tex

```diff
@@ -44,183 +40,377 @@ \section{Introduction}
 \subsection{Objectives}
 \label{sect:objectives}
 
-Testing for the raw image from the DAQ to being written to the archive service at the LDF
-This test activity demonstrates the successful execution of the major components of the storing of raw image files from the DAQ to the DBB and then visualized through the LSST science platform (LSP).   This is at a very small scale and it to test the components of a proof of concept that they all work together.  
+Testing for the raw image from the DAQ to being written to the archive
+service at the LDF\\
+This test activity demonstrates the successful execution of the major
+components of the storing of raw image files from the DAQ to the DBB and
+then visualized through the LSST science platform (LSP). ~ This is at a
```

Comment:
Why `~`?

Reply:
This is an extra space in the Jira LVV-P10 Objective field. That pandoc render as a ~. If this extra space is not desided, has to be removed from LVV-P10.

Reply:
I tried to find as many extra spaces that I could find.